temporian/implementation/numpy/operators/window/base.py

```diff
@@ -62,11 +63,24 @@ def __call__(
             if index_key in input.data:
                 input_data = input.data[index_key]
 
+                window_length_data = None
+                if window_length is not None:
+                    window_length_data = window_length.data[index_key].features[
+                        0
+                    ]
+                    # Check all window length values are positive
+                    if not np.all(window_length_data > 0):
+                        raise ValueError(
+                            "All values in `window_length` must be strictly"
+                            " positive."
+                        )
```

Comment:
It could be dangerous for the data to be able to trigger an error.
Instead, I think that it would be better to have a defined non-error behavior in case the window length is negative (possibly with a warning).

Reply:
Agreed, changed to warning and added test